### PR TITLE
Update changelong for 4.0.0 release

### PR DESCRIPTION
**Pull request by kjsanger**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,15 +6,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+
+## [4.0.0]
+
 ### Added
 
+#### Recipes
+ - bambi 0.14.0
```
Comment (jmtcsngr) [inline CHANGELOG.md:15]: I think baton 2.1.0 also came in with this change

```suggestion
 - bambi 0.14.0
 - baton 2.1.0
```

Comment (kjsanger) [reply CHANGELOG.md:15]: No, baton 2.1.0 is over a year old and was in the previous release.

Comment (kjsanger) [reply CHANGELOG.md:15]: It is missing baton 3.0.0, though.

Comment (kjsanger) [reply CHANGELOG.md:15]: Which I've added.

**Pull request merged**